frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -561,7 +561,7 @@ class SharingSidebar extends React.Component {
       const slackSpec = formInput.channels.slack;
 
       return (
-        <Sidebar onCancel={() => true}>
+        <Sidebar onCancel={onCancel}>
```

Comment:
👍 threading it through like this makes sense.

Separately, we might be able to consolidate the separate but identical `<Sidebar>` uses here, but that's outside of this PR.